corehq/apps/app_manager/static/app_manager/js/apps_base.js

```diff
@@ -19,4 +19,16 @@ $(function () {
     if (v2) {
         hqLayout.utils.setIsAppbuilderResizing(true);
     }
+
+    $(document).ajaxComplete(function(e, xhr, options) {
+        if (/edit_form_attr/.test(options.url) ||
```

Comment:
any other ajax call on the page would trigger this callback, so we'd check the version regardless if it were a save or not. (may naively trigger an infinite loop, the call to check the version will trigger this callback method again)